modules/cert-manager-remove-resources-console.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * security/cert-manager-operator-uninstall.adoc
+
+[id="cert-manager-remove-resources-console_{context}"]
+= Removing {cert-manager-operator} resources
+
+After uninstalling the {cert-manager-operator}, you can remove its related resources from your cluster.
+
+.Prerequisites
+
+* You have access to the cluster with `cluster-admin` privileges.
+* You have access to the {product-title} web console.
+
+.Procedure
+
+. Log in to the {product-title} web console.
+. Remove the `openshift-cert-manager-operator ` namespace.
+.. Navigate to *Administration* -> *Namespaces*.
+.. Click the Options menu {kebab} next to the *openshift-cert-manager-operator* and select *Delete Namespace*.
+.. In the confirmation dialog, enter `openshift-cert-manager-operator` in the field and click *Delete*.
+// . TODO: remove CRDs?
```

Comment:
@slaskawi After we uninstall the cert-manager operator, what residual resources are left that we should tell users to clean up? I have the namespace done here, but should we also have them remove CRDs, the operator group, anything else?

Reply:
I'm not sure.

When uninstalling the operator, OLM leaves the CRDs in the cluster. If the user wishes to remove the operator once and for all, he should probably remove the CRs and CRDs as well. But this is something related to how you uninstall OLM-based operators in general. We're not doing anything special here.